pkg/algorithm/nodeselect/v1alpha1/select_node.go

```diff
@@ -53,11 +55,52 @@ func (ac *Config) NodeBlockDeviceSelector() (*nodeBlockDevice, error) {
 	if err != nil {
 		return nil, err
 	}
-	selectedBD := ac.selectNode(nodeBlockDeviceMap)
+
+	if ProvisioningType(ac.Spc) == ProvisioningTypeAuto {
+		filteredNodeBDs, err = ac.getFilteredNodeBlockDevices(nodeBlockDeviceMap)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	selectedBD := ac.selectNode(filteredNodeBDs)
```

Comment:
in the case of 'Manual', this `filteredNodeBDs` will not be populated

Reply:
Yes just now I found it while fixing the UT

Reply:
While debugging UT failures some how I deleted it

Reply:
Updated